### ISSUE TITLE
Error: data.aws_caller_identity.current: resource repeated multiple times

### ISSUE THREAD

**Issue by josoroma-zz** [FormidableLabs/aws-lambda-serverless-reference#25]
Hi!

Using latest master, I'm hitting:

```
NODE_ENV=development STAGE=development SERVICE_NAME=scbawstfsls yarn run cf:bootstrap:create
```

wait until creation is completed.

```
NODE_ENV=development STAGE=development SERVICE_NAME=scbawstfsls yarn run cf:bootstrap:status
```

then when I run:

```
NODE_ENV=development STAGE=development SERVICE_NAME=scbawstfsls yarn run tf:service:init
```

something goes wrong:

![image](https://user-images.githubusercontent.com/69951/61144131-45d04100-a491-11e9-9e04-82da01d666d6.png)

![image](https://user-images.githubusercontent.com/69951/61143997-f1c55c80-a490-11e9-8705-c07562936c39.png)

```
"env": "echo export NODE_ENV=${NODE_ENV:-localdev}; echo export STAGE=${STAGE:-localdev}; echo export SERVICE_NAME=${SERVICE_NAME:-scbservice}; echo export AWS_REGION=${AWS_REGION:-us-east-1}; echo export AWS_XRAY_CONTEXT_MISSING=LOG_ERROR",
```

Thanks, awesome work!

**Comment by tptee** [comment]
Yikes! Looking into a fix now!

**Comment by tptee** [comment]
Have the fix, just testing fresh creation/deletion!

**Comment by josoroma-zz** [comment]
ooooh @tptee, you're awesome!

**Comment by tptee** [comment]
Fixed in #26! Thanks for catching this!

**Comment by josoroma-zz** [comment]
Thanks a bunch @tptee, you save my day dude!
